This tutorial worked fine, but some of the text needed updates. (#1875)
substrate-developer-hub/substrate-docs: content/md/en/docs/tutorials/get-started/build-local-blockchain.md

   git clone https://github.com/substrate-developer-hub/substrate-node-template
   ```

>    This will put you into the `main` branch. If you are experienced with Polkadot, and want to work with a specific Polkadot version, you can use the `--branch` command-line option to select the Substrate branch that you want to work with.

Reviewer: ```suggestion
   This command clones the `main` branch. 
   If you are familiar with Polkadot, and want to work with a specific Polkadot version, you can use the `--branch` command-line option to select the specific branch that you want to work with.
```
Typically, we want to avoid indefinite articles and future tense. If you specify a branch in this repo it is really a **node template** branch that is compatible with a specific **Polkadot** version. (The tags are all polkadot releases.)

Author: OK.  My goal is to just change the language to be less....optional.  Especially for a beginner tutorial, every time a person has to make a decision about things that they may not fully understand the ramifications for, it is a place where people will stumble, and we will lose some of them. I like your changes to my wording. Thank you.

   cd substrate-node-template
   ```

   Create a new branch to contain your work:

   ```bash
   git switch -c my-learning-branch-yyyy-mm-dd
   ```

   Replace `yyyy-mm-dd` with any identifying information that you desire, but we recommend a numerical year-month-day format. For example:

   ```bash
   git switch -c my-learning-branch-2023-03-01
   ```

3. Compile the node template by running the following command:

   ```bash
   cargo build --release
   ```

>    You should always use the `--release` flag to build optimized artifacts. Be aware that the first time that you do this, it will take a while to complete. Go stretch your legs and get a cup of tea.

Reviewer: ```suggestion
   You should always use the `--release` flag to build optimized artifacts. 
   The first time you compile this, it takes some time to complete.
```

   By default, this option also deletes all active data—such as keys, the blockchain database, and networking information when you stop the node by pressing Control-c.
   Using the `--dev` option ensures that you have a clean working state any time you stop and restart the node.

> 2. Verify your node is up and running successfully by reviewing the output displayed in the terminal.

Reviewer: just fyi that 1. does auto-numbering, so you shouldn't need to change step numbers.

Author: Yeah. There was just inconsistent use of that feature in the tutorial. The `1.` repetition was done in one place, but not others, so I just manually numbered everything when I made my other changes. The auto-numbering feature is cool, and IMHO, it'd probably be better to change all of the ordered lists to just all consistently use that feature, but I figure that can be a totally separate PR just for that.

   node --version
   ```

>    If you have `node` installed, this command will return a version number like `v19.7.0`.

Reviewer: ```suggestion
   If you have `node` installed, this command returns a version number like `v19.7.0`.
```

To use the front-end template:

1. Verify that your current working directory is the root directory where you installed the front-end template in the previous section.

1. Start the front-end template by running the following command:

   ```bash
   yarn start
   ```

> 1. Open `http://localhost:8000` in a browser to view the front-end template. This step may occur automatically when you run the `yarn start` command.

Reviewer: ```suggestion
   Typically, running the `yarn start` command opens `http://localhost:8000` in your default browser automatically.
   If necessary, you can manually specify the `http://localhost:8000` URL to view the front-end template.
```
